usr.sbin/notifyd/Makefile

```diff
@@ -13,6 +13,7 @@ MK_MAN= no
 
 BASE_INCLUDE=  -I. -I${.CURDIR} -I${.CURDIR}/init -I${.CURDIR}/../../include/apple -I${MACHINE_INCLUDES}
 BASE_INCLUDE+= -I${.CURDIR}/../../include -I${.CURDIR}/../../sys -I${.CURDIR}/../../lib/libnotify
+BASE_INCLUDE+= -I${SRCTOP}/include -I/usr/src/include -I${.CURDIR}/../../include #one of these does the trick for in-tree build when missing mach includes
```

Comment:
Please remove this bit `-I/usr/src/include -I${.CURDIR}/../../include #one of these does the trick for in-tree build when missing mach includes`... If `SRCTOP` worked for libdispatch it should also work here. There is already `-I${.CURDIR}/../../include` in the line above, and we don't want hard-coded absolute paths.
Thanks!

Reply:
Yes, I will later, for now my fresh build runs so I can verify the effects of the changes.

Reply:
And have patience with me. The whole PR and stuff still doesn't do exactly what I want it to.